core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -630,47 +635,49 @@ def on_main_gesture(self, event_obj):
+        l2_manager.set_animating(True)
 
         def start_layer2_animation():
             try:
-                Layer2Manager.with_lvgl_timer_pause(
-                    display.cover_background_animate_to_y, -800, 200  # type: ignore[is unknown]
+                l2_manager.with_lvgl_timer_pause(
+                    display.cover_background_animate_to_y,
+                    LAYER2_HIDDEN_Y,
+                    LAYER2_ANIM_TIME_MS,
                 )
             except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)